app/controllers/api/results_controller.rb

```diff
@@ -6,6 +36,50 @@ def results_search_conditions
       MiqReportResult.for_user(User.current_user).where_clause.ast
     end
 
+    def apply_limit_and_offset(results, options)
+      results.slice(options['offset'].to_i, options['limit'].to_i) || []
```

Comment:
it is not, it came form `MiqReportResult#BinaryBlob#data` and also we html_details (MiqReportResult#MiqReportResultDetails) - but we are storing only html. It would be nice for API reasons to store also unformatted data - but on other hand it would be third place where would be report results stored - so I am not sure about this solution